WTWSMS/events/ACR_celtic_flavor_events.txt

```diff
@@ -409,6 +436,7 @@ character_event = {
 		}
 		hidden_tooltip = {
 			character_event = { id = ancrel.0210 days = 10 }
+			remove_trait = traveling
```

Comment:
No, it is a decision in which you get a series of events about travelling through the realm. It is only used there.